### ISSUE TITLE
Feature Ideas for Interactive Mode

### ISSUE THREAD

**Issue by tobias0409** [holxsam/tldraw-in-obsidian#71]
Hey, my feature requests are:

1. Interactive Mode Options: Provide a toggle or menu option to switch between a "Read-Only" mode and a "Full Editor" experience in Interactive Mode. This would allow for quick, inline sketches or adjustments.
2. Persistent Interactive Mode Bounds: Ensure that any changes made to the bounds in Interactive Mode are saved persistently. When reopening a note, it should retain the previously set bounds. - Maybe as optional parameters (x, y, width, height) to the embed code to define the viewport dimensions explicitly.

Thanks for considering.

**Comment by jon-dez** [comment]
1. Do-able, I can implement this in a future release
2. :grinning: no worries haha
3. #59 provides a section "Using `<embed_size>`"
  - For example, if you use `0x100` the height of the embed will be 100 pixels tall
  - Maybe this could be made easier by adding a height adjustment bar?
4. Do-able, future release

Thoughts?

**Comment by tobias0409** [comment]
Hi, thanks for the quick response!

Including points 1 and 4 in a future release sounds fantastic, thank you!

Apologies if I’m mistaken, but I noticed that when the embed reaches its maximum width, it doesn’t scale any further. Even though the max width is already reached, the embed should still expand in height. It would be great to have a height adjustment bar that allows real-time scaling.

Sometimes I write quite a lot, so having the option to insert a larger embed would make it easier to display the entire exercise in a clear, readable size.

**Comment by tobias0409** [comment]
Sure, here is the exported image created in tldraw as an example:
![grafik](https://github.com/user-attachments/assets/78b0eead-defa-40f0-9aaf-b36f683bbfe1)


**What i wish:**
![grafik](https://github.com/user-attachments/assets/2201a835-30ed-48c2-8a93-82fedf8a1cfe)
In this example, you can see that the height expands, making everything readable. However, it becomes too long for the screenshot. :) 

---

**How it currently looks like:**
![grafik](https://github.com/user-attachments/assets/7c266b74-d0a2-4522-ba26-040e7ee83d86)
![grafik](https://github.com/user-attachments/assets/b6f7620f-8b2e-4e04-870d-2950effddf8d)

When embedding the image like this, the "embed image component" has a fixed maximum height. I can either "zoom" with the crop tool (not ideal, as I want to display everything as shown in the image) or scale it by size, as in the current example. However, in this case, some elements, such as X5, X6, etc., are cut off. If I don't scale them by size, the elements are too small to be readable. Note that both images (embeds) here have the same height.


- To summarise:: What I want is for them to grow dynamically in height.

I hope that clarifies what I meant. Thank you very much!

**Comment by jon-dez** [comment]
@tobias0409 

Sorry, I was busy.

Is this what you mean?

https://github.com/user-attachments/assets/880a35e7-eb7a-45b0-a7f3-df72ca67ed5e

**Comment by tobias0409** [comment]
@jon-dez Hey, this is great and exactly what I was looking for! Thanks for your work.
